.travis.yml

```diff
@@ -39,15 +39,21 @@ env:
 
 before_install:
   - gem install slather --no-rdoc --no-ri --no-document --quiet
+  - npm install -g markdown-link-check
 
 script:
 - set -o pipefail
 
+- if [ $CHECK_MARKDOWN == "YES" ]; then
+      echo -en 'travis_fold:start:script.1\\r';
```

Comment:
It prints out a big block with ticks and crosses next to each URL, added a fold so it doesn't take over the entire job window.